ee/server/calendar/google_calendar.go

```diff
@@ -376,17 +386,19 @@ func (c *GoogleCalendar) GetAndUpdateEvent(event *fleet.CalendarEvent, genBodyFn
 		}
 	}
 
-	// If event was deleted/cancelled, we need to stop watching it
-	if !channelStopped {
```

Comment:
why are we no longer watching here?

Reply:
Since the event is recreated, we continue watching the same channel without creating a new channel. This is a speculative fix to improve callback latency issues I've seen.